finance/api/controllers/PaymentController.js

```diff
@@ -0,0 +1,108 @@
+const database = require('../models')
+
+class PaymentController{
+
+    static async createPayment (req, res){
+        const newPayment = {...req.body }
+        const { id, status } = await database.Payments.create(newPayment)
+
+        try {
+            const newPaymentObjectLinks = {
+                id: id,
+                status: status,
+                links: [
+                    {
+                        rel: 'SELF',
+                        method: 'GET',
+                        href: `http://localhost:3010/admin/payments/${id}`
```

Comment:
Seria interessante que o host e a porta fossem dinâmicos neste ponto, caso haja alguma alteração nos valores quando a aplicação for implantada em produção.